config/projects/chefdk.rb

```diff
@@ -97,7 +102,7 @@
   signing_identity "Developer ID Installer: Chef Software, Inc. (EU3VF8YLX2)"
 end
 
-package :msi do
+package :fastmsi do
```

Comment:
@chefsalim, let's leave this as msi for now since a chef-dk release is about to out this week. After that release, let's change the default. In the meantime, we can do builds with just this override.